core/0.18.0/Dockerfile

```diff
@@ -8,20 +8,20 @@ RUN set -ex \
 	&& rm -rf /var/lib/apt/lists/*
 
 ENV BITCOIN_VERSION 0.18.0
-ENV BITCOIN_URL https://bitcoincore.org/bin/bitcoin-core-0.18.0/bitcoin-0.18.0-x86_64-linux-gnu.tar.gz
-ENV BITCOIN_SHA256 5146ac5310133fbb01439666131588006543ab5364435b748ddfc95a8cb8d63f
```

Comment:
By pinning the sha you ensure that upstream can't switch out the signature (and potentially binary) at a later point in time

Reply:
I see. I updated to keep the pinned SHA256 but still it should be checked in the SHA256SUMS.asc otherwise we are just verifying a valid sig and not that it signs the binary.